templates/CRM/MembershipExtras/Page/CurrentPeriodTab.tpl

```diff
@@ -22,17 +64,18 @@
   {foreach from=$lineItems item='currentItem'}
     {assign var='subTotal' value=$subtotal+$currentItem.line_total}
     {assign var='taxTotal' value=$taxTotal+$currentItem.tax_amount}
-
-    <tr id="lineitem-{$currentItem.id}" data-action="cancel" class="crm-entity {cycle values="odd-row,even-row"}">
+    <tr id="lineitem-{$currentItem.id}" data-action="cancel"
+        class="crm-entity rc-line-item {cycle values="odd-row,even-row"}">
       <td>{$currentItem.label}</td>
       <td>{$currentItem.start_date|date_format}</td>
       <td>{$currentItem.end_date|date_format}</td>
-      <td><input type="checkbox" disabled{if $currentItem.auto_renew} checked{/if} /></td>
+      <td><input type="checkbox"
+                 disabled{if $currentItem.auto_renew} checked{/if} /></td>
       <td>{$currentItem.financial_type}</td>
       <td>{$currentItem.tax_amount|crmMoney}</td>
       <td>{$currentItem.line_total|crmMoney}</td>
       <td>
-        <a class="delete" href="">
+        <a class="remove-line-button" href="#" itemid="{$currentItem.id}">

```

Comment:
Can you pls use 'data-itemid' for the attribute name instead of 'itemid' as this will be invalid HTML
Then use jquery .data() to get the attribute
Thanks